pilot/pkg/xds/ads.go

```diff
@@ -446,6 +448,11 @@ func (s *DiscoveryServer) shouldRespond(con *Connection, request *discovery.Disc
 			log.Infof("ADS:%s: FORCE RESPONSE %s for warming.", stype, con.conID)
 			return true, emptyResourceDelta
 		}
+		if features.FilterGatewayClusterConfig && con.proxy.Type == model.Router && request.TypeUrl == v3.ClusterType {
+			log.Infof("ADS:%s: FORCE RESPONSE %s for any pending rds because of cluster warming.",
```

Comment:
I am confused how  returning "should respond to CDS" implies anything about RDS?